.github/workflows/build_documentation.yml

```diff
@@ -0,0 +1,52 @@
+name: build_documentation
+on: [push, pull_request, workflow_dispatch]
+permissions:
+    contents: write
+
+jobs:
+  build_jammy_humble:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    container:
+      image: ghcr.io/bdaiinstitute/spot_ros2_jammy_humble:main
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install package
+        run: |
+          source /opt/ros/$ROS_DISTRO/setup.bash
+          # pull ros_utilites and spot_wrapper
+          # git submodule init 
+          # git submodule update
+          pwd 
+          ls
+          apt-get update
+          apt install wget 
+          ./install_spot_ros2.sh
+          # go to root folder
+          cd ../../
+          colcon build --symlink-install
+          source install/local_setup.bash
+      - name: Install dependencies & build docs
+        run: |
+          source /opt/ros/$ROS_DISTRO/setup.bash
+          python -m pip install --upgrade pip
+          pip install -r docs/requirements.txt
+          sudo pip3 install transforms3d
```

Comment:
- Standarize pip vs pip3
- No reason to have separate pip installs here vs just rolling into requirements.txt?
- Any reason not to move everything into `.devcontainer/Dockerfile`?

Reply:
I've put everything in docs/requirements.txt

Reply:
ROS2 make this complicated. 

If this is a pure Python file, you can define the core dependencies and dev dependencies separately (in either requirements.txt or pyproject.toml), which serves a different purpose than the Dockerfile (though you could still install them through the list when you build the image).

Here, ROS2 says the actual core dependencies should be in [package.xml](https://github.com/bdaiinstitute/spot_ros2/blob/main/spot_driver/package.xml) BUT in practice it's the Dockerfile that is easier to reference, so people treat the Dockerfile as a stand in.

This is a nit. Let's stick to your solution.